### PR TITLE
New datatype: SBML

### PR DESCRIPTION
**Pull request by cmonjeau**
SBML is a free and open interchange format for computer models of biological processes.

### CODE REVIEW

File: lib/galaxy/datatypes/xml.py [hunk 1]
```diff
@@ -159,3 +159,25 @@ def sniff_prefix(self, file_prefix):
...
         """
         return file_prefix.search(OWL_MARKER)
+
+
+class Sbml(GenericXml):
+    """
+        System Biology Markup Language
+        http://sbml.org
+    """
+    file_ext = "sbml"
```
Comment (bgruening) [inline lib/galaxy/datatypes/xml.py:170]: Thats super cool! Can you please also add the edam terms to it? http://bioportal.bioontology.org/ontologies/EDAM/?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Fformat_2585&jump_to_nav=true

Comment (cmonjeau) [reply lib/galaxy/datatypes/xml.py:170]: Done :)

Comment (bgruening) [reply lib/galaxy/datatypes/xml.py:170]: No edam_data available? :)

**Comment by bgruening** [comment]
@cmonjeau thanks!

**Pull request merged**